config.json

```diff
@@ -4,6 +4,17 @@
   "blurb": "",
   "checklist_issue": 3,
   "exercises": [
+    {
+      "slug": "hello-world",
+      "uuid": "6c88f46b-5acb-4fae-a6ec-b48ae3f8168f",
```

Comment:
This uses the same UUID as the C# one. Each exercise's UUID must be globally unique though. You can generate a new v4 UUID at https://www.uuidgenerator.net/version4